BlueprintUI/Sources/Layout/LayoutSubelement.swift

```diff
@@ -0,0 +1,8 @@
+import CoreGraphics
+
+
+public typealias LayoutSubelements = [LayoutSubelement]
```

Comment:
If this is how it stays, I suppose we should ignore my previous note.

Reply:
The `typealias` leaves the door open for us to add a custom type if we ever want to, but I think it's unlikely.